src/Ratchet/Session/SessionProvider.php

```diff
@@ -198,7 +198,7 @@ private function parseCookie($cookie, $host = null, $path = null, $decode = fals
         $pieces = array_filter(array_map('trim', explode(';', $cookie)));
 
         // The name of the cookie (first kvp) must include an equal sign.
-        if (empty($pieces) || !strpos($pieces[0], '=')) {
+        if (empty($pieces) || !strpos(strval($pieces[0]), '=')) {
```

Comment:
seems unnecessary, because the result of `explode` is always an array of strings.